### ISSUE TITLE
add support for stupidpenisx.com (alias for vxtwitter.com/fixvx.com)

### ISSUE THREAD

**Issue by GalaxinTM** [imputnet/cobalt#727]
### service name

stupidpenisx.com

### service description

it's an alias for fixvx, an embed helper for twitter/x posts on discord
should be treated just like normal twitter/x links

### link samples

```shell
https://stupidpenisx.com/ChiefofIsaac/status/1833271162463543469
https://stupidpenisx.com/hourly_shitpost/status/1827375390685323355
https://stupidpenisx.com/shitpost_2077/status/1827050674619658542
https://stupidpenisx.com/weirddalle/status/1816900499628523950
```

**Comment by wukko** [comment]
this is an alias to fxtwitter, which we don't support for moral reasons

**Comment by GalaxinTM** [comment]
shit it just got acquired by fxtwitter around September 23rd - 26th, I swear it was vxtwitter when I opened this up I'm not tripping
![image](https://github.com/user-attachments/assets/783089ff-6d0a-4dc7-8d38-a0036ad0cba8)

**Comment by wonderlarr** [comment]
Hello, I'm the owner of this website (stupidpenisx.com), I host it cause I think it's funny and so do my friends. I literally just use a 309 redirect to redirect all requests to fxtwitter. It was vxtwitter but I switched it at request of a friend, I'm going to switch it back now. 

If someone could enlighten me, what is the difference between vx and fx? I don't know of any history or controversy.

**Comment by GalaxinTM** [comment]
there's a whole thread about it in bluesky, and someone in the comments provided the doc
https://bsky.app/profile/wukko.me/post/3l73ay4yage2p
tl;dr doc claims that fxtwitter's owner is a groomer

**Comment by wonderlarr** [comment]
> there's a whole thread about it in bluesky, and someone in the comments provided the doc https://bsky.app/profile/wukko.me/post/3l73ay4yage2p tl;dr doc claims that fxtwitter's owner is a groomer

After going through that, yeah, will not be using that anymore. Thanks for the info! I didn't open this issue but at this point would you consider adding support for this domain?
